### PR TITLE
[IMP] l10n_pk: Added list of states of pakistan

### PR DESCRIPTION
**Pull request by hsal-odoo**
The 7 states of pakistan were added in English language.

The states of pakistan were missing.

task-3777571

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by hsal-odoo** [comment]
> Per task 3644762, unless Pakistan Post recommends or requires using the state name in postal address please put the information in a localisation module instead.
> 
> Also per task 3644762 it might be a better idea to put both latin and urdu names in the same value similar to the autonomous communities of spain rather than duplicate them, but @bouvyd should probably be the judge of that bit.

@xmo-odoo the same approach was followed here https://github.com/odoo/odoo/pull/128243

**Comment by RicGR98** [comment]
A unrelated test is failing on runbot, could you rebase ?

File: addons/l10n_pk/data/res.country.state.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"id","country_id/id","name","code"
+state_pk_ajk,base.pk,"آزاد جموں و کشمیر","AJK"
+state_pk_ba,base.pk,"بلوچستان","BA"
+state_pk_gb,base.pk,"گلگت بلتستان","GB"
+state_pk_is,base.pk,"اسلام آباد دار الحکومت","IS/ICT"
+state_pk_kp,base.pk,"خیبر پختونخوا","KP/KPK"
+state_pk_pb,base.pk,"پنجاب","PB"
+state_pk_sd,base.pk,"سندھ","SD"
```
Comment (malb-odoo) [inline addons/l10n_pk/data/res.country.state.csv:8]: Missing EOF, set up you IDE to do it for you :smile:

**Review by malb-odoo — CHANGES_REQUESTED** [review]
Good job ! Small thing here and it will be good :smile:

**Review by malb-odoo — APPROVED** [review]
LGTM thanks for that ! keep the nice work :smile: It needs a rebase thought

**Comment by qdp-odoo** [comment]
@robodoo r+

@hsal-odoo thank you :+1:

**Pull request closed without merging**